### DISCUSSION TITLE
Using plugin with swift in Bazel Apple rules and swift package manager

### DISCUSSION THREAD

**Discussion by richieapple** [noctule-ide/noctule-swift-plugin#34] [General]
I’m just started a project where I use swift with bazel on Apple MacOs 

I would like to use clion to edit swift code. 

I can jenerate compile command json 

What is the best way to get this plugin working ?

**Comment by jansorg** [comment]
The plugin uses swift LSP, which still needs a "swift build" to provide most of its output, including completions.
Unfortunately, the plugin does not yet manage the build on its own. Is your bazel build doing something similar compared to swift build, e.g. are you seeing completions after running a full build of ypur swift project?
